PCbuild/prepare_ssl.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python3
+# -*- coding: utf8 -*-      for <2.7>/PC/VS9.0/build_ssl.py
```

Comment:
prepare_ssl.py is (currently) only used when pushing a new OpenSSL to the externals repo.  Its purportedly to be run by python3 (see shebang), as all Python (Windows) releases use the same externals.

I generally prefer inline comments for "notes", especially compared to *following* comments, but not strongly enough to hold up this inclusion.